scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -4,10 +4,10 @@ import {SCHEDULED_OUTPUT, DESK_OUTPUT} from 'apps/desks/constants';
 import {appConfig} from 'appConfig';
 
 AggregateCtrl.$inject = ['$scope', 'desks', 'workspaces', 'preferencesService', 'storage',
-    'savedSearch'];
+    'savedSearch', 'content'];
```

Comment:
Can you add TypeScript types to scope and places where you changed things? It would also help if you could add a few tests since this view has long gotten messier than it should be.